satellite/payments/stripecoinpayments/version.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package stripecoinpayments
+
+import (
+	"context"
+	"time"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/internal/sync2"
+)
+
+// ErrVersion defines version service error.
+var ErrVersion = errs.Class("version service error")
+
+// VersionService updates conversion rates in a loop.
```

Comment:
We have to tag type with `architecture: Service`, but ask @egonelbre because I'm not entirely sure.